offchain/advance-runner/src/config.rs

```diff
@@ -38,6 +37,21 @@ pub struct AdvanceRunnerConfig {
     pub backoff_max_elapsed_duration: Duration,
 }
 
+#[derive(Debug, Clone, Parser)]
+pub struct RunnerHealthCheckConfig {
```

Comment:
Apply these changes to the host-runner as well. (It is in the main branch.)

Reply:
I don't understand what exactly I have to do. Do I initiate another server for health check for the host runner? Or do I add a health endpoint to the http server? Do I rename the config file? There's a lot I can do on the host runner.

Reply:
Since the host-runner should only be used during DApp development, what are the benefits of adding the health check to it?

Reply:
> I don't understand what exactly I have to do. Do I initiate another server for health check for the host runner? Or do I add a health endpoint to the http server? Do I rename the config file? There's a lot I can do on the host runner.

We need to standardize how we set up these HTTP servers. Since the health check is only part of the problem, we can do that in another issue.